src/main/res/values/themes.xml

```diff
@@ -45,7 +45,6 @@
         <item name="android:actionBarStyle">@style/Theme.Amahi.ActionBar</item>
         <item name="android:windowBackground">@color/background_primary</item>
         <item name="android:textColorPrimaryInverse">@color/primary_text_material_light</item>
-        <item name="android:tint">@color/accent</item>
```

Comment:
Why did you remove this statement?

Reply:
I removed this statement because it was overriding the color of the up button. Instead, I used fillColor property to overcome this situation. I am going to run this in an emulator running Android 5.0, 6.0, and 8.1 and add a screenshot if it works on those versions.

Reply:
I feel it's better to keep android:tint property so that uniformity is maintained. In case we plan to change tint in the future, we can change that with a single line change. Further, we won't have to explicitly write fillColor in all drawable files.

Reply:
What I think is first, we are just using android: tint property just in Settings Activity and nowhere other in the app and It is causing undesirable behavior and second, we can now also change color of all icons through changing just color of accent in colors.xml file. I have attached screenshots in my comment below proving it works on all android versions not less than API 21.

Reply:
What do you mean by "undesirable behavior"?

Reply:
I mean that showing different color than expected.